autodeploy/docker-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cd
+git clone https://github.com/TiK15Ryyb/Approbatur.app.git
```

Comment:
If the repo would be private, here we would get through it using a Github access token, for example (now this is enough)

Reply:
Indeed, good point. Let's ride on the waves of public repos here ;)